src/core/common/DashboardCard/index.js

```diff
@@ -63,9 +64,9 @@ export default props => {
     return (
         <Grid item xs={12} sm={6} md={6} lg={3}>
             <Card className={classes.card}>
-                <div className={classes.topBlock} onClick={() => history.push('/' + list)}>
+                <div id={id} className={classes.topBlock} onClick={() => history.push('/' + list)}>
```

Comment:
it's weird that you need an id in react. What is this one for?

Reply:
It is necessary for UserTour, because it will search elements on the table by element ID.

Reply:
👍